airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3296,9 +3302,24 @@ components:
           description: The query parameters present in the redirect URL after a user granted consent e.g auth code
           type: object
           additionalProperties: true # Oauth parameters like code, state, etc.. will be different per API so we don't specify them in advance
-    CompleteOauthResponse:
+        connectionConfiguration:
+          $ref: "#/components/schemas/DestinationConfiguration"
+    CompleteSourceOauthResponse:
       type: object
-      additionalProperties: true # Oauth parameters like refresh/access token etc.. will be different per API so we don't specify them in advance
+      properties:
+        oauthOutput:
+          type: object
+          additionalProperties: true # Oauth parameters like refresh/access token etc.. will be different per API so we don't specify them in advance
```

Comment:
I would expect this requires some small tweak to do in the front-end code to keep it compatible with the API change?
(the completeOauthResponse is now nested under `oauthOutput`)

cc @Jamakase